modules/commandModule/commandWorker_flight.py

```diff
@@ -1,4 +1,8 @@
-from modules.commandModule.commandFns import write_pigo
+from modules.commandModule.commandModule import CommandModule
+from modules.commandModule.commandFns import write_pigo, read_pogi
+
+POGI_DIR = ""
+PIGO_DIR = ""
 
 def flight_command_worker(pipelineIn, pipelineOut):
 	# pipelineIn gives [[x,y], [range]]
```

Comment:
Out of scope but could you please put the logic here (lines 4-18) inside a while loop. Then implement the changes in my following comment and call `pogi_subworker(...)` here, passing in the pipeline out and directory.

Reply:
sorry...not too sure what you mean. do you mean put the pogi and pigo logic into one while loop in one worker function? if so, then would you like me to write up a `pigo_subworker(...)` as well?